exonum-node/src/lib.rs

```diff
@@ -935,10 +932,10 @@ impl NodeChannel {
     /// Returns the channel for sending timeouts, networks and API requests.
     fn node_sender(&self) -> NodeSender {
         NodeSender {
-            internal_requests: self.internal_requests.0.clone().wait(),
-            network_requests: self.network_requests.0.clone().wait(),
-            transactions: self.transactions.0.clone().wait(),
-            api_requests: self.api_requests.0.clone().wait(),
+            internal_requests: SyncSender::new(self.internal_requests.0.clone()),
```

Comment:
Do we need to continue to use sync sender version in the node sender?
Now we can just spawn future in the any place of code.

Reply:
I'll try to encapsulate spawning within `SyncSender` so as not to break the existing usage. I guess this will require conditional compilation for sandbox tests or some other hack to get spawning work outside of `tokio` context 🙃